datastore.yml

```diff
@@ -0,0 +1,16 @@
+---
+- hosts: datastore-dev
+  remote_user: root
+  gather_facts: no
+  roles:
```

Comment:
Does this want the `config` role to be run?

Reply:
Nope.

Reply:
nope